consensus/src/main/java/org/apache/iotdb/consensus/ratis/Utils.java

```diff
@@ -69,13 +69,18 @@ public static String groupFullName(ConsensusGroupId consensusGroupId) {
     return String.format("%s-%d", groupTypeAbbr, consensusGroupId.getId());
   }
 
-  public static RaftPeer toRaftPeer(Endpoint endpoint) {
+  // priority is used as ordinal of leader election
+  public static RaftPeer toRaftPeer(Endpoint endpoint, int priority) {
     String Id = String.format("%s-%d", endpoint.getIp(), endpoint.getPort());
-    return RaftPeer.newBuilder().setId(Id).setAddress(IP_PORT(endpoint)).build();
+    return RaftPeer.newBuilder()
+        .setId(Id)
+        .setAddress(IP_PORT(endpoint))
```

Comment:
Change IP_PORT to ipAddress. Id is IP-Port and address is IP:Port, thus has a little difference. I use the slash instead of colon because the ratis won't allow a : appears in the naming scheme.